build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/should-transform-jss-var/output.mjs

```diff
@@ -1,4 +1,5 @@
-var _classes = JSON.parse("{\"button\":\"button-21aa4a8\",\"CSS\":\".button-21aa4a8{font-size:12px}\\n\"}");
+/** @enum {string}*/
+var _classes = {"button":"button-21aa4a8"};
```

Comment:
Are we allowed to use `""` on enum keys?

Reply:
Yep. See the example I posted above

Reply:
Oh wait...this may not work. The example is broken with quoted keys but works when unquoted